src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -104,23 +97,22 @@ export default {
             expanded: true,
             parentKeyString: this.openmct.objects.makeKeyString(this.domainObject.identifier),
             conditionCollection: [],
+            conditionResults: {},
             conditions: [],
             currentConditionIdentifier: this.currentConditionIdentifier || {},
-            telemetryObjs: this.telemetryObjs,
+            telemetryObjs: [],
             moveIndex: Number,
             isDragging: false
         };
     },
     destroyed() {
-        this.composition.off('add', this.addTelemetry);
+        this.composition.off('add', this.addTelemetryObject);
     },
     mounted() {
-        this.telemetryObjs = [];
-        this.conditionResults = {};
         this.instantiate = this.openmct.$injector.get('instantiate');
         this.composition = this.openmct.composition.get(this.domainObject);
-        this.composition.on('add', this.addTelemetry);
-        this.composition.on('remove', this.removeTelemetry);
+        this.composition.on('add', this.addTelemetryObject);
+        this.composition.on('remove', this.removeTelemetryObject);
```

Comment:
Need a corresponding `this.composition.off` for `'remove'` in the when the view is destroyed.

Reply:
done